content/dpg indicators/8/Open Hardware/_index.en.adoc

```diff
@@ -33,4 +32,4 @@ Special thanks for the information in this category goes to the following contri
 * https://sfconservancy.org/about/staff/#deb[Nicholson, Deb]
 * Urra, Jose
 * Velis, Emilio
-* Wagenknecht, Addie
+* Wagenknecht, Addie
```

Comment:
Same comment here: the URL path should not include whitespace and should use all lowercase (i.e. `/8/open-hardware/`).

Reply:
Addressed in commit 0eabde03d98eda5e3fb0debea1bfa5781f8440e3.